code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -197,6 +197,20 @@
 	desc = "<I>Cantharellus Cibarius</I>: These jolly yellow little shrooms sure look tasty!"
 	icon_state = "chanterelle"
 
+/obj/item/food/grown/mushroom/chanterelle/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/kitchen/spoon))
+		return ..()
+	if(seed.potency < 95)
+		return ..()
+
+	to_chat(user, span_notice("You hollow up the chanterelle with [I]."))
+	remove_item_from_storage(user)
+	qdel(src)
+	if(seed.resistance_flags & FIRE_PROOF)
+		user.put_in_hands(/obj/item/clothing/head/wizard/chanterelle/fr)
+	else
+		user.put_in_hands(/obj/item/clothing/head/wizard/chanterelle)
```

Comment:
this doesn't work. you aren't passing in objects you're passing in typepaths, so put_in_hands will immediately runtime. you need to create these and then pass them in, as we do above

Reply:
what about now?

Reply:
works but put some () at the end of the typepath for consistency's sake